src/patternfly/components/Button/button.scss

```diff
@@ -316,6 +332,66 @@
     }
   }
 
+  &.pf-m-control {
+    position: relative;
+    border: none;
+
+    &::before {
```

Comment:
any reason you're using `::before` instead of `::after`? Seems like you would get most of the definitions for free, then you can just redefine `--pf-c-button--BorderWidth`. Then I might separate `border: var(--pf-c-button--BorderWidth) solid var(--pf-c-button--BorderColor);` into `border: var(--pf-c-button--BorderWidth) solid;` and `border-color: var(--pf-c-button--BorderColor);`, then redefine `--pf-c-button--BorderColor` as

Reply:
Ah the reason I used before instead of after was because I based it off the clipboard-copy-group-copy, but after definitely makes sense in this case. Updated.